Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM solr:7.4-alpine
+
+# Do some setup as root.
+USER root
+
+# Install curl.
+RUN apk update && apk add curl
+
+# Work as the solr user.
+USER solr
+
+# Work from the base dir of the Solr install.
+WORKDIR /opt/solr/
+
+# Anserini JAR for TwitterAnalyzer.
+COPY --chown=solr target/anserini-0.2.1-SNAPSHOT-fatjar.jar lib/
```

Comment:
We probably want a change (or copy) the final fatjar artifact to something with a constant name (i.e., `anserini.jar`).